.azure/pipelines/ci.yml

```diff
@@ -19,6 +38,8 @@ jobs:
   parameters:
     # Ensures the alignment of branch name and deployment params
     buildArgs: '/warnaserror:BUILD1001'
+    ${{ if or(eq(variables['Build.Reason'], 'Schedule'), and(eq(variables['System.TeamProject'], 'internal'), eq(variables['Build.Reason'], 'Manual'), eq(parameters.runCodeQL3000, 'true'))) }}:
+      runCodeQL3000Tasks: true
```

Comment:
Name change is inconsistent. `runCodeQL3000Tasks` here and in default-build.yml but `runCodeQL3000` in project-ci.yml (the layer in between). Suggest going w/ `runCodeQL3000` everywhere.